rustls/src/error/mod.rs

```diff
@@ -154,6 +154,31 @@ pub enum Error {
     Other(OtherError),
 }
 
+/// Determine which alert should be sent for a given error.
+///
+/// If this mapping fails, no alert is sent.
+impl TryFrom<&Error> for AlertDescription {
+    type Error = ();
+
+    fn try_from(error: &Error) -> Result<Self, Self::Error> {
+        match error {
```

Comment:
Nit: alternatively, maybe hoist the `Ok(..)` and yeet the `Err(())` directly by explicitly `return`ing?